### ISSUE TITLE
Rule for not allowing prop next to component opening tag in case of multi-line props

### ISSUE THREAD

**Issue by jseminck** [yannickcr/eslint-plugin-react#534]
Is there a rule which does not allow to have a property right next to the Component opening tag in the case of multline property declaration?

Allowed:
```javascript
<MyComponent myProp="hello" />
```

Allowed:
```javascript
<MyComponent
  myProp="hello"
  foo="bar"
/>
```

Not allowed (because once we have a property on a new line, then the first prop should also be on a new line.
```javascript
<MyComponent myProp="hello"
  foo="bar"
/>
```

If not, if there is a request for such a rule I would not mind to look into it.

Thanks.

**Comment by petersendidit** [comment]
Seems like the [jsx-first-prop-new-line.md ](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-first-prop-new-line.md) rule with the `multiline` option does exactly this.

**Comment by yannickcr** [comment]
Yep, solved by https://github.com/yannickcr/eslint-plugin-react/pull/535
